sonar-db/src/test/resources/org/sonar/db/version/v60/MakeUuidColumnNotNullOnSnapshotsTest/in_progress_snapshots.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE "SNAPSHOTS" (
+  "ID" INTEGER NOT NULL GENERATED BY DEFAULT AS IDENTITY (START WITH 1, INCREMENT BY 1),
+  // NULLABLE
```

Comment:
good point. It works on h2 (see http://www.h2database.com/html/grammar.html#comments) but I will still change it.